spec/convertion_by_format/check_open_docx_by_screen_spec.rb

```diff
@@ -13,7 +13,7 @@
       pending 'https://bugzilla.onlyoffice.com/show_bug.cgi?id=32793' if file_path == 'docx/Office Open XML Part 4 - Markup Language Reference.docx'
       pending 'https://bugzilla.onlyoffice.com/show_bug.cgi?id=42324' if file_path == 'docx/SUMMER_APPLICATION_PG_1.docx'
       s3.download_file_by_name('docx/' + File.basename(file_path), './files_tmp')
-      link = "#{StaticData.nginx_url}/#{File.basename(file_path)}"
+      link = "#{StaticData.nginx_url}/#{FileHelper.file_rename(File.basename(file_path))}"
```

Comment:
This line is very complex. Can you to divide file renaming and link creating on 2 lines of code

Reply:
Maybe this should be extracted to method

Reply:
```

Reply:
Isn't such a solution enough?
`tmp_name = FileHelper.file_rename(File.basename(file_path))`
`link = "#{StaticData.nginx_url}/#{tmp_name}"`

Reply:
I think, it's enough